docs/en/connector-v2/source/Hive.md

```diff
@@ -180,6 +180,71 @@ Description:
 - `kerberos_keytab_path`: The keytab file path for Kerberos authentication.
 - `krb5_path`: The path to the `krb5.conf` file used for Kerberos authentication.
 
+Run the case:
+
+```bash
+env {
+  parallelism = 1
+  job.mode = "BATCH"
+}
+
+source {
+  Hive {
+    table_name = "default.test_hive_sink_on_hdfs_with_kerberos"
+    metastore_uri = "thrift://metastore:9083"
+    hive.hadoop.conf-path = "/tmp/hadoop"
+    result_table_name = hive_source
```

Comment:
This pr break #8153 test case on https://github.com/apache/seatunnel/actions/runs/12132734926/job/33827217564.
Could you create a PR to fix it? @zhangshenghang